resilience4j-framework-common/src/main/java/io/github/resilience4j/common/circuitbreaker/configuration/CommonCircuitBreakerConfigurationProperties.java

```diff
@@ -727,6 +735,10 @@ public Double getRandomizedWaitFactor() {
         }
 
         public InstanceProperties setRandomizedWaitFactor(Double randomizedWaitFactor) {
+            if (randomizedWaitFactor < 0 || randomizedWaitFactor >= 1) {
```

Comment:
Checked during circuit breaker work [here](https://github.com/resilience4j/resilience4j/blob/b81dd07aaa10a8f69fa983ed03e14f062048c552/resilience4j-core/src/main/java/io/github/resilience4j/core/IntervalFunction.java#L253). Not sure why 1 is not allowed.

Reply:
I don't remember as well :(